src/components/App.tsx

```diff
@@ -14,10 +17,13 @@ import { State } from '../state/reducer'
 function App() {
   const shouldGoToLandingPage = useSelector(({ ui }: State) => !ui.skipLandingPage)
   const initialQueryString = window.location.search
-
+  let history = useHistory()
   return (
     <Switch>
-      {shouldGoToLandingPage && <Redirect exact from="/" to="/start" />}
+      {shouldGoToLandingPage &&
```

Comment:
I think we can refactor to the use "to" prop in the Redirect component rather than invoking a function which causes a side effect inside the rendering of this component. See react router documentation: https://reacttraining.com/react-router/core/api/Redirect/to-object

Reply:
Yup, make sense. I think the original one doesn't work is because it ignores the search, will give it a try tonight!